<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1631</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>모델 관리 프로그램 v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>1133</width>
      <height>471</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>상세</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이름</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LIST 추가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>생년월일</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>연락처</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>인스타</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>작성일자</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>파일명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>파일위치</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이미지경로</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>22</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#424242;&quot;&gt;모델 PROFILE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>181</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_prev">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(67, 67, 67);</string>
       </property>
       <property name="text">
        <string>◀</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_page">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>26</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_next">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(67, 67, 67);</string>
       </property>
       <property name="text">
        <string>▶</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_profile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>242</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_profile">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_profile">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>9</y>
       <width>50</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_profile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>85</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_career">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_career">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_career">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>9</y>
       <width>50</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_career">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>85</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>22</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#424242;&quot;&gt;경력(CAREER)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_contact">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_contact">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_contact">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>9</y>
       <width>50</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_contact">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>85</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>514</x>
      <y>22</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#424242;&quot;&gt;매니저/소속사&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_contract">
    <property name="geometry">
     <rect>
      <x>756</x>
      <y>40</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_contract">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_contract">
     <property name="geometry">
      <rect>
       <x>184</x>
       <y>9</y>
       <width>50</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_contract">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>85</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>759</x>
      <y>22</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#424242;&quot;&gt;계약 정보&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>조  회</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbox_input_item">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>1133</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_items_clear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_other">
    <property name="geometry">
     <rect>
      <x>1001</x>
      <y>40</y>
      <width>152</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit_other">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_other">
     <property name="geometry">
      <rect>
       <x>93</x>
       <y>9</y>
       <width>50</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1004</x>
      <y>22</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#424242;&quot;&gt;기타 정보&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>신  규</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_select_list">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>20</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>모델명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>삭제</string>
     </property>
    </column>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>0</y>
      <width>20</width>
      <height>731</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1284</x>
      <y>179</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/arrow_down.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget_images">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>210</y>
      <width>231</width>
      <height>461</height>
     </rect>
    </property>
    <attribute name="headerVisible">
     <bool>true</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1631</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>조회</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource/model_images.qrc"/>
 </resources>
 <connections/>
</ui>
